nixos/modules/flyingcircus/packages/all-packages.nix

```diff
@@ -163,7 +163,7 @@ in rec {
   postfix = pkgs.callPackage ./postfix/3.0.nix { };
   powerdns = pkgs.callPackage ./powerdns.nix { };
 
-  prometheus = pkgs_17_03.prometheus;
+  prometheus = pkgs_17_09.prometheus;
```

Comment:
We already have included a 17_09 branch in our main, right?

Reply:
yes